src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -232,8 +234,31 @@ const PagedTable: FC<Props> = ({result, properties}) => {
     setTotalPages,
   } = useContext(PaginationContext)
   const [height, setHeight] = useState(0)
+  const [headerHeight, setHeaderHeight] = useState(0)
+  const [rowHeight, setRowHeight] = useState(0)
   const ref = useRef()
 
+  if (headerHeight === 0) {
+    const calculatedHeaderHeight =
+      document.querySelector<HTMLElement>(`#${pagedTableHeaderId}`)
+        ?.offsetHeight ?? 0
```

Comment:
Something to consider when using `querySelector` vs `ref`s: https://stackoverflow.com/a/69260293/92446. Not sure how feasible it is to use refs here, or if one is better than the other.

Reply:
Yeah, `useRef` is easier. i wouldn't need `ids` at all even though this calculation is done only once. It's a minor improvement, but still an improvement!